packages/integration-tests/src/client/index.ts

```diff
@@ -24,18 +33,29 @@ export default class MockClient {
 
   private navigateUrl?: string;
   private readonly api: Got;
+  private readonly interactionApi: Got;
 
-  constructor(config?: Partial<LogtoConfig>) {
+  constructor(
+    config?: Partial<LogtoConfig>,
+    interactionApi?: Got,
+    /* Skip ID token JWT verification */
+    skipIdTokenVerification = false
+  ) {
     this.storage = new MemoryStorage();
     this.config = { ...defaultConfig, ...config };
-    this.api = got.extend({ prefixUrl: this.config.endpoint + '/api' });
-
-    this.logto = new LogtoClient(this.config, {
-      navigate: (url: string) => {
-        this.navigateUrl = url;
+    this.interactionApi = interactionApi ?? got;
+    this.api = this.interactionApi.extend({ prefixUrl: this.config.endpoint + '/api' });
+
+    this.logto = new LogtoClient(
+      this.config,
+      {
+        navigate: (url: string) => {
+          this.navigateUrl = url;
+        },
+        storage: this.storage,
       },
-      storage: this.storage,
-    });
+      skipIdTokenVerification ? (client) => new EmptyJwtVerifier(client) : undefined
```

Comment:
nit: could leave some comments here. take advantage of the buildJwtVerifier callback to skip the idTokenVerification. 

Also, instead of passing through a bool `skipIdTokenVerification` value, maybe directly allow an optional `buildJwtVerifier` is a better option.

Reply:
So the client build parameters can be aligned with our SDK.